content/millennium/dstu2/general-clinical/condition.md

```diff
@@ -7,6 +7,29 @@ title: Condition | DSTU 2 API
 * TOC
 {:toc}
 
+## Overview
+The Condition Resource is used to record details about a patient's problems, diagnoses, or other health matters that are concerning. It is common to capture Conditions during the encounter or visit - both as initial suspected problems and confirmed or refuted problems or diagnoses at the time of discharge. This resource may be referenced by other resources as a "reason" for an action or order such as the reason for ordering a medication or procedure.
+
+This resource is NOT intended to record subjective or objective information that could lead to the recording of a condition such as signs or symptoms. Signs and symptoms are typically documented as Observations although some persistent symptoms such as fever or headache may be documented prior to the definitive diagnosis being recognized by a clinician.
+
+References to implicitRules and modifierExtensions are NOT supported and will fail a Create or Update request.
```

Comment:
Should be consistent though. If we wrap these with `` there are other fields (abatementDateTime for example) which should also be wrapped.

That might start to lean more toward being noise rather than being helpful.